packages/functions/transform/src/merge-request-metrics.ts

```diff
@@ -955,13 +971,25 @@ export async function run(extractMergeRequestId: number, ctx: RunContext) {
   })
     .get();
 
+  const { id: transformSourceBranchId } = await upsertBranch(ctx.transformDatabase, {
+    name: extractData.mergeRequest.sourceBranch || "",
+  })
+    .get();
+
+  const { id: transformTargetBranchId } = await upsertBranch(ctx.transformDatabase, {
+    name: extractData.mergeRequest.targetBranch || "",
+  })
+    .get();
+
```

Comment:
Issue: Branches can not have names, maybe for deleted branches or something ? How do we treat null branch